coalib/processes/Processing.py

```diff
@@ -303,7 +309,15 @@ def filter_raising_callables(it, exception, *args, debug=False, **kwargs):
     """
     for elem in it:
         try:
+            if debug_bears is False:
+                kwargs['debug_flag'] = False
+            elif ((str(debug_bears[0]).lower().strip())=='true') or \
+                ((elem.__name__) in debug_bears):
+                kwargs['debug_flag'] = True
+            else:
+                kwargs['debug_flag'] = False
```

Comment:
you can compress this `if` to a ternary `if`, since two of the branches are the same.